tests/WE2E/machine_suites/fundamental.cheyenne

```diff
@@ -0,0 +1,9 @@
+grid_RRFS_CONUScompact_13km_ics_HRRR_lbcs_RAP_suite_HRRR
+grid_RRFS_CONUS_25km_ics_NAM_lbcs_NAM_suite_HRRR
+specify_RESTART_INTERVAL
+specify_template_filenames
+grid_SUBCONUS_Ind_3km_ics_HRRR_lbcs_RAP_suite_HRRR
+grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_RAP_suite_RRFS_v1beta
+pregen_grid_orog_sfc_climo
+custom_GFDLgrid__GFDLgrid_USE_NUM_CELLS_IN_FILENAMES_eq_FALSE
+custom_GFDLgrid__GFDLgrid_USE_NUM_CELLS_IN_FILENAMES_eq_TRUE
```

Comment:
These added machine_suite files seem outside the scope of NCO compliance. What am I missing?